src/main/scala/com.socrata.geoexport/encoders/ShapefileEncoder.scala

```diff
@@ -226,6 +228,18 @@ object ShapefileEncoder extends GeoEncoder {
             zipStream.putNextEntry(new ZipEntry(file.getName))
             IOUtils.copy(fis, zipStream)
             zipStream.closeEntry()
+            // I cannot for the life of me figure out if it's possible
+            // to get geotools to emit a shapefile.  No combination of
+            // setting an encoding and/or setting the "try cpg file"
+            // setting seems to do it.  So, hackhackhack.
+            if(file.getName.endsWith(".dbf")) {
```

Comment:
Newb question: what's a .dbf file?

Reply:
The main way shapefiles are weird is that they're basically a list of shapes (a "shp" file) containing the shapefile's one geometry column stapled (by being packaged in the same zip or directory) to a dbase file (the "dbf" file) that contains all the other data.